jupyterhub_moss/templates/option_form.html

```diff
@@ -190,6 +198,25 @@ <h4 style="text-align: center">List of available resources:</h4>
       placeholder="hh:mm:ss"
       pattern="[0-9]+:[0-5][0-9]:[0-5][0-9]"
     />
+    <label for="environment" accesskey="e">
+      Jupyter environment:
+    </label>
+    <select
+      name="environment"
+      id="environment"
+    >
+      <option id="environment_custom" value="">Custom...</option>
+    </select>
+    <input
+      type="text"
+      class="form-offset-row"
+      id="environment_path"
+      name="environment_path"
+      placeholder="/path/to/jupyter/environment/bin"
+    />
+    <p id="environment_path_note" class="form-offset-row" hidden>
+      &#9888; The <a href="https://pypi.org/project/batchspawner/">batchspawner</a> package must be installed in this environment.
```

Comment:
Won't this pose problem if the version of `batchspawner` does not match the one we use ?